test/integration/narrow/helpers/models.test.js

```diff
@@ -0,0 +1,111 @@
+describe('Models', () => {
+  const question = {
+    type: 'mock_type',
+    backUrl: 'mock_back_url',
+    key: 'mock_key',
+    answers: [{
+      value: 'mock_answer_value',
+      hint: 'hint',
+      text: 'answer_text'
+    }],
+    backUrlObject: {
+      dependentQuestionYarKey: 'tenancyLength',
+      dependentAnswerKeysArray: ['tenancy-length-A1'],
+      urlOptions: {
+        thenUrl: 'tenancy-length',
+        elseUrl: 'tenancy-length-condition',
+        nonDependentUrl: 'tenancy'
+      }
+    },
+    sidebar: {
+      values: [{
+        heading: 'Eligibility',
+        content: [{
+          para: `This grant is for pig, beef or dairy farmers.
+          
+          Poultry, arable-only, contractors and horticultural growers are not currently eligible.`
+        }]
+      }]
+    },
+    score: '123',
+    warning: {
+      text: 'Other types of business may be supported in future schemes',
+      iconFallbackText: 'Warning'
+    }
+  }
+
+  test('getModel', () => {
+    const { getModel } = require('../../../../app/helpers/models')
```

Comment:
This line should be added to the beginning of the file instead. That way, it does not need to be added for each individual test